AI/Nullkiller/Pathfinding/AINodeStorage.cpp

```diff
@@ -968,14 +966,13 @@ struct TowmPortalFinder
 		std::vector<const CGTownInstance *> targetTowns,
 		AINodeStorage * nodeStorage)
 		:actor(actor), initialNodes(initialNodes), hero(actor->hero),
-		targetTowns(targetTowns), nodeStorage(nodeStorage)
+		targetTowns(targetTowns), nodeStorage(nodeStorage),
+		spellID(SpellID::TOWN_PORTAL), townPortal(spellID.toSpell()), movementNeeded(GameConstants::BASE_MOVEMENT_COST)
 	{
-		spellID = SpellID::TOWN_PORTAL;
-		townPortal = spellID.toSpell();
 
 		// TODO: Copy/Paste from TownPortalMechanics
 		townPortalSkillLevel = SecSkillLevel::SecSkillLevel(hero->getSpellSchoolLevel(townPortal));
-		movementNeeded = GameConstants::BASE_MOVEMENT_COST * (townPortalSkillLevel >= SecSkillLevel::EXPERT ? 2 : 3);
+		movementNeeded *=  townPortalSkillLevel >= SecSkillLevel::EXPERT ? 2 : 3;
```

Comment:
Weird change, old code was more explicit & self-describing

Reply:
@IvanSavenko old code was constantly reordered by clang-tidy into non-buildable mess. I will think how to change it a good way.